modules/table/src/main/java/org/apache/ignite/internal/table/distributed/replicator/LeaderOrTxState.java

```diff
@@ -20,35 +20,34 @@
 import java.io.Serializable;
 import org.apache.ignite.internal.tx.TxMeta;
 import org.apache.ignite.internal.tx.message.TxStateReplicaRequest;
-import org.apache.ignite.network.ClusterNode;
 import org.jetbrains.annotations.Nullable;
 
 /**
- * Response for the {@link TxStateReplicaRequest}. Can contain either the Partition Group leader, which should be
+ * Response for the {@link TxStateReplicaRequest}. Can contain either the consistent ID of the Partition Group leader, which should be
  * queried for the TX Meta, or the TX Meta itself.
  */
 public class LeaderOrTxState implements Serializable {
     private static final long serialVersionUID = -3555591755828355117L;
 
     @Nullable
-    private final ClusterNode leader;
+    private final String leaderName;
```

Comment:
Why do not use consistent id instead of name here?

Reply:
Name and consistent ID are the same thing (I know, this ambiguity is very misleading)